### ISSUE TITLE
Bug when disabling Sentry 

### ISSUE THREAD

**Issue by YoannMa** [hydra-newmedia/hapi-sentry#22]
The update adding the new client options cause some issue when I want to disable sentry (in CI/CD env)

```javascript
const Hapi = require('@hapi/hapi');

const run = async () => {
    const server = Hapi.server();

    await server.register({
        plugin  : require('hapi-sentry'),
        options : {
            client : {
                dsn : false
            }
        }
    });

    server.route({
        method: 'GET',
        path:'/',
        handler: () => 'Hello World!'
    });

    await server.inject({ url : '/' });
};

run();
```

It seems that it doesn't create the instance of Sentry.
And because of that, in the request extension point, it try to create a Scope from the config object.

My current workaround is create my own Sentry client with `Sentry.init({ dsn : false })`.

**Comment by fiws** [comment]
Sorry for that. We released a fix in v1.2.1

**Comment by YoannMa** [comment]
Thanks for the quick fix :rocket: !
